storagenode/peer.go

```diff
@@ -244,6 +244,9 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config Config, ver
 func (peer *Peer) Run(ctx context.Context) error {
 	group, ctx := errgroup.WithContext(ctx)
 
+	group.Go(func() error {
```

Comment:
The order of these shouldn't matter since these are goroutines that are started at the same time.